### ISSUE TITLE
Question about fine-tuning

### ISSUE THREAD

**Issue by Show-han** [GitGyun/visual_token_matching#13]
After reading the paper, it appears that you mentioned using 10 images as support data for fine-tuning the bias. However, upon reviewing your code, I noticed that fine-tuning was conducted on more than 10 images. Could you clarify this inconsistency? Additionally, could you explain the purpose of the released fine-tuning checkpoints and specify the dataset on which they were fine-tuned? 🤔

**Comment by GitGyun** [comment]
I guess you have confused about the `dset_size` argument and `shot` argument. The `dset_size` of FinetuneDataset has been set to be more than 10 to control the number of iterations per epoch (due to using the pytorch lightning). Inside the `__getitem__` of the FinetuneDataset, you can check that only `shot` number of data are indexed.
Also, the fine-tuned checkpoint for each fold are fine-tuned with the specified task in the checkpoint name, which is used to report the few-shot learning performance.

**Comment by Show-han** [comment]
> I guess you have confused about the `dset_size` argument and `shot` argument. The `dset_size` of FinetuneDataset has been set to be more than 10 to control the number of iterations per epoch (due to using the pytorch lightning). Inside the `__getitem__` of the FinetuneDataset, you can check that only `shot` number of data are indexed. Also, the fine-tuned checkpoint for each fold are fine-tuned with the specified task in the checkpoint name, which is used to report the few-shot learning performance.

Thanks for your reply.
So, can I understand it this way:
When fine-tuning, you use just 10 images but repeat them several times within one epoch. Then, during testing, you use the same 10 images as the support set. Are these 10 images fixed based on the random seed or the meta_info/*.pth? (I'm not certain if this is correct.) 
Additionally, the results you reported in Table 1 are solely based on the 10 selected support images. You use these 10 support images for fine-tuning, and also as the support set when testing all images in the test split. Is my understanding accurate?

**Comment by GitGyun** [comment]
1. Support images are fixed by the pre-defined permutation indices stored in `meta_info` directory.
2. Yes, the selected support images are both used for fine-tuning and testing, and no other images are used during the stages.

**Comment by Show-han** [comment]
Thank you very much for your reply! Hope I can contribute a citation to your work🤣
